tests/restarting/to_6.2.33/CycleTestRestart-1.txt

```diff
@@ -28,3 +28,6 @@ testTitle=Clogged
     testName=SaveAndKill
     restartInfoLocation=simfdb/restartInfo.ini
     testDuration=10.0
+
+storageEngineExcludeTypes=2,3
```

Comment:
why does this PR need this change?

Reply:
It's part of the changes that we need to fix downgrade testing